### ISSUE TITLE
Manage kinds != [1, 6, 30023]

### ISSUE THREAD

**Issue by dtonon** [mikedilger/gossip#553]
Currently Gossip shows not short/long kinds (ex. 1311, Live Chat Message) without any context, and this can be quite confusing.
We should manage them somehow. Options:

A) Hide them;
B) Show a refer, for example adding a  "from Live stream" linked to a client that support it (ideally njump.me);

Example:

<img width="778" alt="image" src="https://github.com/mikedilger/gossip/assets/89577423/64ba29f2-a25a-4432-92bd-576be5980af1">

https://njump.me/nevent1qqs2mulefwsv43flswus7meszlt9kv7fxsn2g9jaffrw24ng9lv3rpgppemhxue69uhkummn9ekx7mp0qyghwumn8ghj7mn0wd68ytnhd9hx2tczu90h5

**Comment by mikedilger** [comment]
I recently updated nostr-types list of events that are displayable.  Gossip then subscribes to all events that are displayable. We can change this at a higher level, but it is important to do it at the right level (previously nostr-types was saying they weren't even a displayable kind).  I'm not sure exactly what to do.  Discussion?

**Comment by dtonon** [comment]
I would filter them at the app level, if possibile.
This could be also useful if we add a configurable filter by kinds to the feed (or to the lists).

**Comment by bu5hm4nn** [comment]
Each of those chat entries is tied to a live event which is a kind 30311 entry. So I think we should show the 30311 in the feed, later we can give it a special styling so it's clear it's a special event.
Then I think we should create a separate FeedKind for live event chat that includes all the 1311, and then display them as a thread and later as a chat, once we succeeded in creating a reverse chronological ScrollView.

So basically you see the 30311 in your feed and when you click the "Thread" button it will pop over to a chat view that displays all the 1311 (and lets you participate eventually).

**Comment by mikedilger** [comment]
Let's not implement NIP-53 right now we have plenty else to do. Let's just remove this from the feed.

**Comment by mikedilger** [comment]
Fixed on unstable. Open another issue if you want to argue for supporting any of these kinds, one issue per kind.

ChannelMessage
LiveChatMessage
CommunityPost
DraftLongFormContent
